CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-04-16
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
```

Comment:
Install OpenCost on AKS https://techcommunity.microsoft.com/t5/apps-on-azure-blog/leverage-opencost-on-azure-kubernetes-service-to-understand-and/ba-p/3796813 
OpenCost has released a special KubeCon EU 2023 build incorporating the above features along with additional community member contributions. To try the build, refer to [Azure OpenCost Integration](https://www.opencost.io/docs/azure-opencost). These features will be generally available in production in OpenCost v103, scheduled for May 2, 2023.